torchdynamo/config.py

```diff
@@ -62,7 +62,7 @@
 dynamic_propagation = True
 
 # Run the FX graph with FakeTensors
-fake_tensor_propagation = True
+fake_tensor_propagation = False
```

Comment:
?

Reply:
Oh it lost my older comment - this is for easier prototyping atm, not going to land as is.

Reply:
<img width="824" alt="image" src="https://user-images.githubusercontent.com/4755252/188518632-cc552f77-bda2-43fc-af81-c00f8c6dec95.png">
:P